2.7.0/final.md

```diff
@@ -0,0 +1,1073 @@
+# PyTorch 2.7.0 Release Notes
+- [Highlights](#highlights)
+- [Backwards Incompatible Changes](#backwards-incompatible-changes)
+- [Deprecations](#deprecations)
+- [New Features](#new-features)
+- [Improvements](#improvements)
+- [Bug fixes](#bug-fixes)
+- [Performance](#performance)
+- [Documentation](#documentation)
+- [Developers](#developers)
+
+# Highlights
+TODO @seemethere
+
+For more details about these highlighted features, you can look at the release blogpost.
+Below are the full release notes for this release.
+
+# Backwards Incompatible Changes
+
+### Dropped support for Triton < 2.2.0. Removed Support for CUDA 12.4, Anaconda in CI/CD.
+- Removed CUDA 12.4 support in CI/CD in favor of 12.8 (#148895, #142856, #144118, #145566, #145844, #148602, #143076, #148717)
+- Removed Anaconda support in CI/CD (#144870, #145015, #147792)
+- Dropped support for Triton < 2.2.0 (versions without ASTSource) (#143817)
+
+### Change `torch.Tensor.new_tensor()` to be on the given Tensor's device by default ([#144958](https://github.com/pytorch/pytorch/pull/144958))
+
+This function was always creating the new Tensor on the "cpu" device and will now use the same device as the current Tensor object. This behavior is now consistent with other `.new_*` methods.
+
+### Use Manylinux 2.28 and CXX11_ABI=1 for future released Linux wheel builds. 
+With Migration to manylinux_2_28 (AlmaLinux 8 based), we can no longer support OS distros with glibc2_26. These include popular Amazon Linux 2 and CentOS 7. (#143423, ##146200, #148028, #148135, #148195, #148129)
+
+### `torch.onnx.dynamo_export` now uses the ExportedProgram logic path (#137296)
+
+Users using the `torch.onnx.dynamo_export` API may see some `ExportOptions` become
+unsupported due to an internal switch to use `torch.onnx.export(..., dynamo=True)`: `diagnostic_options`, `fake_context` and `onnx_registry` are removed/ignored by `ExportOptions`. Only `dynamic_shapes` is retained.
+
+Users should move to use the `dynamo=True` option on `torch.onnx.export` as
+`torch.onnx.dynamo_export` is now deprecated. Leverage the [`dynamic_shapes`](https://pytorch.org/docs/stable/export.html#torch.export.export) argument in `torch.onnx.export` for specifying dynamic shapes on the model.
+
+Version 2.6.0
+
+```py
+torch.onnx.dynamo_export(model, *args, **kwargs)
+```
+
+Version 2.7.0
+
+```py
+torch.onnx.export(model, args, kwargs=kwargs, dynamo=True)
+```
+
+### Finish deprecation of `LRScheduler.print_lr()` along with the `verbose` kwarg to the LRScheduler constructor. (#147301)
+
+Both APIs have been deprecated since 2.2. Please use `LRScheduler.get_last_lr()` to access the learning rate instead.`print_lr` and `verbose` were confusing, not properly documented and were little used, as described in #99270, so we deprecated them in 2.2. Now, we complete the deprecation by removing them completely. To access and print the learning rate of a LRScheduler:
+
+Version 2.6.0
+```
```

Comment:
these code blocks and a few others are missing the "python" marker to get proper syntax highlighting